src/chrome/content/rules/Krisostomus.xml

```diff
@@ -1,30 +1,40 @@
 <!--
-	Problematic domains:
+	Breaks search by subject on www.kriso.ee
+		http://www.kriso.ee/health-personal-development-su-V-english2.html
+		http://www.kriso.ee/language-su-C-english2.html
+		http://www.kriso.ee/technology-engineering-agriculture-su-T-english2.html
```

Comment:
I copied the comment from #11200 but maybe it's simpler to label it as `Different content http/https`.